lib/py/src/transport/TSSLSocket.py

```diff
@@ -266,6 +266,10 @@ def __init__(self, host='localhost', port=9090, *args, **kwargs):
         TSocket.TSocket.__init__(self, host, port, unix_socket,
                                  socket_keepalive=socket_keepalive)
 
+    # Override TSocket because ssl.SSLSocket does not allow non-zero flags in calls to recv()
+    def isOpen(self):
+        return self.handle is not None
```

Comment:
I would slightly prefer to just catch the exception in `TSocket.isOpen`, but I don't feel strongly either way.

Reply:
I guess that makes some sense in that we'd largely expect the implementations to be in-sync.